typings/axe-core/axe-core.d.ts

```diff
@@ -0,0 +1,152 @@
+// Type definitions for axe-core 2.0.5
+// Project: https://github.com/dequelabs/axe-core
+// Definitions by: Marcy Sutton <https://github.com/marcysutton>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ElementContext {
+	node?: Object,
+	selector?: string,
+	include?: any[],
+	exclude?: any[]
+}
+interface RunOnly {
+	type: string, // "rule" | "rules" | "tag" | "tags"
+	value?: {
+		include?: string[],
+		exclude?: string[]
+	}
+	values?: string[] // "wcag2a" | "wcag2aa" | "section508" | "best-practice"
+}
+interface AxeResults {
+	url: string,
+	timestamp: string,
+	passes: Pass[],
+	violations: Violation[]
+}
+interface Pass {
+	description: string,
+	help: string,
+	helpUrl: string,
+	id: string,
+	impact: string, // "minor" | "moderate" | "serious" | "critical" | "null"
```

Comment:
If you use the same strings, you can use a type alias like this:

```ts
export type ImpactValues = "minor | "moderate" | "serious" | "critical" | "null";
```

or something similar. String literal types've been around since 1.8, so you can feel free to use them here too.